api/v1/convert/version.go

```diff
@@ -41,3 +42,23 @@ func GetFixedBy(vStr string, vuln *validation.CVESchema) (string, error) {
 
 	return "", nil
 }
+
+// GetFixedByIstioVuln gets the fixed-by version for vStr in Istion vuln.
+func GetFixedByIstioVuln(vStr string, vuln types.Vuln) (string, error) {
```

Comment:
This and `func isAffected(vStr string, vuln types.Vuln) bool` in `istio/cache/db.go` are very similar. I'm thinking maybe we can make a utility package in `pkg` and inside there, we have something like the following:

```
func IstioIsAffected(v version, vuln types.Vuln) (bool, string, error)
```

which tells you if the given Istio version is affected by the vulnerability, the FixedBy version, if it's affected, and an error, if anything. What do you think?